chess/ChessGameModel.swift

```diff
@@ -18,6 +18,7 @@ final class ChessGameModel: ObservableObject {
 
     private var webSocketManager: WebSocketManager?
     private(set) var player: ChessPlayer?
```

Comment:
let's make them explitly unrwapped optionals here. I'm sure we rarely expect them to be `nil`

Reply:
I need help with this